client/trino-jdbc/src/test/java/io/trino/jdbc/BaseTestJdbcResultSet.java

```diff
@@ -94,7 +95,10 @@ public void testPrimitiveTypes()
             checkRepresentation(connectedStatement.getStatement(), "1.0E0 / 0.0E0", Types.DOUBLE, Double.POSITIVE_INFINITY);
             checkRepresentation(connectedStatement.getStatement(), "0.0E0 / 0.0E0", Types.DOUBLE, Double.NaN);
             checkRepresentation(connectedStatement.getStatement(), "true", Types.BOOLEAN, true);
-            checkRepresentation(connectedStatement.getStatement(), "'hello'", Types.VARCHAR, "hello");
+            checkRepresentation(connectedStatement.getStatement(), "'hello'", Types.VARCHAR, (rs, column) -> {
+                assertEquals(rs.getObject(column), "hello");
+                assertThat(rs.getAsciiStream(column)).hasBinaryContent("hello".getBytes(StandardCharsets.US_ASCII));
```

Comment:
Could you add assertions for null with `getAsciiStream` and `getBinaryStream`?

Reply:
added